src/assets/styles/sections/blog-posts.scss

```diff
@@ -1,7 +1,5 @@
-/* ================ Blog post ================ */
 .blog-post {
   display: flex;
-  flex-direction: row;
   flex-wrap: wrap;
 }
 
```

Comment:
You'll also need to fix the following in both files for the linting to pass:

![image](https://user-images.githubusercontent.com/991693/39444476-f676dab0-4c85-11e8-8867-07e810ee1e93.png)

Essentially, the formatting needs to be `(1 / 3)`.